pages/dkp/kommander/2.1/install/configuration/http-proxy/index.md

```diff
@@ -86,6 +86,12 @@ Kommander installs with a dedicated CLI.
     ./kommander install --installer-config ./install.yaml
     ```
 
+1. Configure the `kommander-flux` namespace and adjust the label so the Gatekeeper mutation is active on the namespace:
```

Comment:
@mhrabovcin  - two steps prior to this one, we instruct the user to create the kommander ns and label it similarly.   Could we combine these two steps (i.e. tell user to pre-create both kommander and kommnder-flux and label them appropriately?  Or must this mutation change occur only *after* install of kommander?    My main concern here is that these two steps are VERY similar with very similar NS names and worry a customer might skip this one thinking they already did it above.   If we (can?) combine both into one step it seems a bit clearer to me.

Reply:
I've merged these to a single step.